source/administration/incident-response-application.rst

```diff
@@ -0,0 +1,667 @@
+Incident Response Application (EE, Closed Alpha)
+------------------------------------------------------
+
+The incident response application is available in closed Alpha, in `Enterprise Edition <https://mattermost.com/pricing/>`_. Supported in Mattermost 5.12 and later.
+
+Use the incident response application to connect all your workflows, automate repetitive tasks and collaborate on incidents within one secure messaging platform. Sample use cases you can accomplish with this app include the following:
+
+1. Trigger automated incident response workflows based on keywords.
+2. Automatically mention your InfoSec or DevSecOps teams when an incident occurs, including via email, mobile push and desktop notifications.
+3. Auto-create "war rooms" and invite key team members to immediately collaborate on a critical incident.
+4. Take quick actions, review data and access relevant links all in one place.
+5. Archive resolved incidents to declutter your channel sidebar without losing access to past information.
+6. Pull summary statistics of the incident response workflow, including mean-time-to-acknowledgment (MTTA) and mean-time-to-resolution (MTTR).
+
+Below is an example of the incident response app automatically creating a channel from a Nagios alert, adding users to the channel and providing them quick actions to take on the incident.
+
+.. image:: ../images/incident-response-app-intro-image.png
+   :alt: Incident Response App: Intro Image
+
+.. image:: ../images/incident-response-app-intro-image-fullpage.png
+   :alt: Incident Response App: Intro Image - Fullpage
+
+To shape the direction of the incident response application, `sign up to our Alpha program here <https://docs.google.com/forms/d/e/1FAIpQLSf4Rr1YnofQQnKHJuL0Cgz_DaCUitt_Atik7K9KXsDefCyXlg/viewform>`_!
+
+.. contents:: Contents
+  :backlinks: top
+  :local:
+  :depth: 2
+
+How Can I Try The App?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. `Sign up for the closed Alpha program <https://docs.google.com/forms/d/e/1FAIpQLSf4Rr1YnofQQnKHJuL0Cgz_DaCUitt_Atik7K9KXsDefCyXlg/viewform>`_. If you're selected to join the Alpha program, you will receive an email from us with a plugin binary.
+
+2. Upload the plugin binary in **System Console > Plugins > Plugin Management** `following these steps <https://about.mattermost.com/default-plugin-uploads>`_.
+
+3. Enable the plugin from the **Installed Plugins** list.
+
+4. Upload an incident response workflow to your Mattermost server:
+
+ - Write an app workflow schema in a JSON file. You can use `one of these sample app workflows <https://github.com/mattermost/docs/tree/master/source/samples/incident-response-app>`_ to get started, and edit the sample (such as team and channel names) with the help of the configuration documentation below.
```

Comment:
FYI - For the link to samples to work, we need https://github.com/mattermost/docs/pull/3113 to be merged